docs/reviewing-release-prs.md

```diff
@@ -0,0 +1,176 @@
+- Adding a new export to a package
+
+However, it is important to remember that consumers care about capabilities first and interfaces second. So, some additions could also be more simply regarded as changes. For instance:
+
+- Adding a new dependency
+
+### Removals
+
+In most cases, removing something from a package will cause a breaking change, and thus should be categorized as such. There are some rare instances, however, when this is not the case:
+
+- Removing a runtime dependency
+
+### Fixes
+
+There are a few different kinds of changes that could be filed under this category.
+
+- Fixing a security vulnerability.
+- Correcting a mistake introduced in a previous release. For example, a function was supposed to be included but was not.
+- Realigning confusing or illogical behavior. This is tricky, because the longer buggy behavior is allowed to persist, the more consumers may rely upon it. In this case, it would be more thoughtful to call out a change in this behavior as breaking, even if the new behavior makes more sense. (As an example, read about the [history of MetaMask's signing methods](https://github.com/MetaMask/metamask-docs/blob/72e1a066ea1a81a7964dbc9aa83381493a2cd3eb/wallet/concepts/signing-methods.md).) But for minor cases, a "fix" may be more appropriate.
+
+### Everything else
+
+These constitute changes which cannot be neatly categorized as breaking, or as an addition, deletion, or fix, such as:
+
+- Adding, upgrading, or removing a runtime dependency
+- Modifying behavior in a low-impact, non-surprising way
+
+## 3. Review new version strings
+
+Now that you understand the changes being made and what sort of impact they are likely to have on consumers, the next step is to verify that the impact is being appropriately communicated via version strings.
+
+[Semantic Versioning](https://semver.org/) (SemVer) dictates that versions should be bumped in the following manner:
+
+- If there are any changes you would categorize as a breaking change, bump the **major** (first) part of the version.
+- Otherwise, if there are any changes you would categorize as an addition, bump the **minor** (second) part of the version.
+- Otherwise, bump the **patch** (third) part of the version.
+
+If there are any packages being bumped whose new versions understate their impact, then notify the creator of the release PR so that they can update the versions appropriately.
+
+## 4. Review existing changelog entries
+
+Next, look over the changelogs in the packages-to-be-released to ensure that they are well formed and that they list all changes that will be released.
+
+The [Keep a Changelog](https://keepachangelog.com/) specification defines a standard format for changelogs, and all MetaMask repositories, including the core monorepo, follow this format.
+
+You might notice that the categories listed in this spec match the same ones listed above in this document. This is not a coincidence!
```

Comment:
Hmmm... the "Breaking" category certainly isn't one of the change entries.

Reply:
Yes, this statement is inaccurate. I can remove it.

Reply:
This is removed.